static/app/views/performance/traceDetails/content.tsx

```diff
@@ -91,6 +92,46 @@ class TraceDetailsContent extends React.Component<Props, State> {
     filteredTransactionIds: undefined,
   };
 
+  componentDidUpdate(prevProps: Props) {
```

Comment:
`componentDidUpdate()` will not be called for the initial render. Should there be something called for `componentDidMount()`?